packages/react-instantsearch/examples/autocomplete/README.md

```diff
@@ -0,0 +1,8 @@
+This example shows how to build an autocomplete with react-instantsearch
+
+To start the example: 
+
+* npm install
+* npm start
+
+Read more about react-instantsearch [in our documentation](https://community.algolia.com/instantsearch.js/react/).
```

Comment:
Comments inside the source code of the example would be better no? And maybe a link back to the guide from the README, WDYT?